README.md

```diff
@@ -117,6 +117,14 @@ cd test_runner
 pytest
 ```
 
+If you see a `Found interfering processes running` error, stop pageserver and postgres nodes for all branches:
```

Comment:
This error message is from the test runner that has its own readme, we can put it there instead: https://github.com/zenithdb/zenith/blob/main/test_runner/README.md

I've been suggested once to do the `killall -9 postmaster pageserver wal_acceptor postgres` to fix the error, we can add the command too.

Reply:
I think it's better to put it directly in the error message, PTAL. I would be wary of `killall`, maybe someone will try to run tests on a non-isolated production server.

The script does not check for `postmaster`, does not?

Reply:
Nice idea about the error message.

The `killall` one does not check for anything, no.